src/Microsoft.AspNetCore.Mvc.Core/Internal/MemoryPoolHttpResponseStreamWriterFactory.cs

```diff
@@ -26,10 +26,10 @@ public class MemoryPoolHttpResponseStreamWriterFactory : IHttpResponseStreamWrit
         /// Creates a new <see cref="MemoryPoolHttpResponseStreamWriterFactory"/>.
         /// </summary>
         /// <param name="bytePool">
-        /// The <see cref="ArrayPool{byte}"/> for creating <see cref="byte"/> buffers.
+        /// The <see cref="ArrayPool{Byte}"/> for creating <see cref="byte"/> buffers.
```

Comment:
Hmm, seeing this a lot. Should we standardize on `T` and match what users will actually read? (Maybe `TKey` and `TValue` for dictionaries.)